File: Homeworks/01. C# Fundamentals/03. Operators-and-Expressions/08. Prime Check/PrimeCheck.cs
    using System;
    class PrimeNumberCheck
    {
        static void Main()
        {
            int number = int.Parse(Console.ReadLine());

            Console.WriteLine(IsPrime(number) ? "true" : "false");
        }

        private static bool IsPrime(int number)
        {
            if (number < 2) return false;
            if (number == 2) return true;
            for (int i = 2; i <= (int)Math.Sqrt(number); i++)
            {
                if (number % i == 0) return false;
            }
            return true;
        }
    }

File: Homeworks/06. JS Fundamentals/04. Operators and Expressions/07. Prime Check/PrimeCheck.js
function PrimeCheck(args) {
    var number = args[0];
    if (number >= 2) {
        for (var i = 2; i <= Math.sqrt(number); i += 1) {
            if (number % i == 0) {
                console.log(false);
                return;
            }
        }
        console.log(true);
        return;
    }
    else {
        console.log(false);
    }
}